RHEL6_7/networking/samba/check

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+. /usr/share/preupgrade/common.sh
+
+#END GENERATED SECTION
+
+
+SOLUTION_FILE="./solution.txt"
+
+echo "SMB1 protocol is deprecated as it is replaced by SMB2 protocol. It is now considered insecure, due to it's vulnerability to man-in-the-middle attacks. Support for the SMB2 protocol will not be backported to Red Hat Enterprise Linux 6 version of samba\* packages. Therefore, for security and compatibility reasons, we strongly recommend to the user to upgrade whole Red Hat Enterprise Linux based server - clients network at once." > $SOLUTION_FILE
```

Comment:
When the message is persistent and there is not any other check than the samba-common package available on system, the message shoud be part of the solution.txt file.